templates/web/bexley/main_nav_items.html

```diff
@@ -1,4 +1,6 @@
 [%~ IF bodyclass.match('waste') OR (problem AND problem.cobrand_data == 'waste') ~%]
+    [% INCLUDE navitem uri='/waste' label='Lookup address' %]
```

Comment:
I think this needs to be 'Look up address' (with space) - or 'Address lookup' (because look up is a verb in the former and a noun in the latter, or something). I can do a quick fix 👍 (Did Bexley ever specify what text they wanted exactly?)

Reply:
Thanks 🙏 . No, I couldn't find any spec for the text they wanted.